ui/raidboss/data/05-shb/raid/e8s.js

```diff
@@ -870,20 +875,20 @@
         'Akh Morn': 'Akh Morn',
         'Absolute Zero': 'Zéro absolu',
 
-        // FIXME
-        'Reflected Frost \\(G\\)': 'Reflected Frost (G)',
-        'Reflected Frost \\(R\\)': 'Reflected Frost (R)',
-        'Reflected Kick \\(G\\)': 'Reflected Kick (G)',
-        'Reflected Wings \\(B\\)': 'Reflected Wings (B)',
-        'Reflected Wings \\(G\\)': 'Reflected Wings (G)',
-        'Reflected Wings \\(R\\)': 'Reflected Wings (R)',
-        'Twin Silence/Stillness': 'Twin Silence/Stillness',
-        'Reflected Armor \\(B\\)': 'Reflected Armor (B)',
-        'Reflected Armor \\(G\\)': 'Reflected Armor (G)',
-        'Reflected Armor \\(R\\)': 'Reflected Armor (R)',
-        'Spiteful/Embittered Dance': 'Spiteful/Embittered Dance',
-        'Reflected Drachen': 'Reflected Drachen',
-        'Inescapable Illumination': 'Inescapable Illumination',
+        // Double check please
```

Comment:
Are you asking me to double check? Or is this a note for future translators?

Reply:
It's for other FR translator, maybe @VanSlanzar can have a look.

Reply:
i look again after, baby need me sorry.